prod/westeurope/linux/appbackendl1/web_test/terragrunt.hcl

```diff
@@ -0,0 +1,33 @@
+# Common
+dependency "application_insights" {
+  config_path = "../../../common/application_insights"
+}
+
+dependency "resource_group" {
+  config_path = "../../../common/resource_group"
+}
+
+dependency "app_service" {
+  config_path = "../app_service"
+}
+
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_application_insights_web_test?ref=v2.1.30"
+}
+
+inputs = {
+  name = "appbackendl1"
+
+  resource_group_name     = dependency.resource_group.outputs.resource_name
+  application_insights_id = dependency.application_insights.outputs.id
+  enabled                 = true
+  geo_locations           = ["emea-nl-ams-azr"]
+
+  url = format("https://%s/info", dependency.app_service.outputs.default_site_hostname)
+}
```

Comment:
What do u think to add a variable to enabled it inside app_service module like function_app?
https://github.com/pagopa/io-infrastructure-modules-new/blob/master/azurerm_function_app/main.tf#L165
https://github.com/pagopa/io-infrastructure-modules-new/blob/master/azurerm_function_app/vars.tf#L167